exonum-java-binding/core/rust/generate_licenses.sh

```diff
@@ -8,13 +8,13 @@ cargo license -h &> /dev/null || cargo install cargo-license
 
 # Going to exonum-java-binding-core/rust directory.
 backup_dir=${PWD}
-script_dir=$(dirname $(readlink -f "$0")) # Directory of this script. Must be exonum-java-binding/core/rust
+script_dir=$(dirname $(realpath -s "$0")) # Directory of this script. Must be exonum-java-binding/core/rust
```

Comment:
`readlink -f` doesn't work on Mac, and `realpath -s` is appeared to be a better solution

Reply:
Perhaps it is worth mentioning that `realpath` should be installed explicitly on Mac (like `brew install coreutils`)

Reply:
Oh, really?) I just had it installed, perhaps I installed coreutils some time ago :)